Short-Answer/Algorithms_Answers.md

```diff
@@ -3,13 +3,26 @@
 ## Exercise I
 
 a)
-
+O(n)
+The loop requires n^3 loops before stopping, but since a is incremented by n^2, it reduces the number of loops by n^2. n^3/n^2 = n
 
 b)
-
+O(n^2)
+The outer loop loops by n and nested by another n loop which results in n^2. n*n = n^2
+This should be less than n^2 because the second n is reduced by j *= 2 considerably
```

Comment:
You are on the right track here. Good job for noticing that the two loops iterate at different rates. So I think you are correct in that it's less than n^2. Think on it some more...